content/docs/cml/self-hosted-runners.md

```diff
@@ -83,27 +83,27 @@ you do it your way :)
-| `--cloud-gpu`            | GPU type.                                                                                                       | Choices: `nogpu`,`k80`,`tesla`                                                                                 |
-| `--cloud-hdd-size`       | HDD size in GB.                                                                                                 | Accepts integer values                                                                                         |
-| `--cloud-ssh-private`    | Your private RSA SSH key. If not provided will be generated by the Terraform-provider-Iterative.                | Accepts string                                                                                                 |
-| `--cloud-spot`           | Request a spot instance                                                                                         | Boolean                                                                                                        |
-| `--cloud-spot-price`     | Spot max price. If not specified it takes current spot bidding pricing.                                         | Default: `-1`                                                                                                  |
-| `--cloud-startup-script` | Run the script provided in-line during the instance initialization. Expects a Base64-encoded Linux shell script | String                                                                                                         |
-| `-h`                     | Show help                                                                                                       | Boolean                                                                                                        |
+| Argument                 | Type                                                                  | Default                                                    | Description                                                                                                                  |
+| ------------------------ | --------------------------------------------------------------------- | ---------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `--version`              | Flag                                                                  | Unset                                                      | Show version number and exit.                                                                                                |
+| `--labels`               | List                                                                  | `cml`                                                      | Comma delimited list of runner labels.                                                                                       |
```

Comment:
TBH this whole "Type" column isn't too convincing. All values passed are strings, technically right? May be best to remove it and specify what kind of values are expected as part of the description if needed.

In this case e.g.

```suggestion
| `--labels`               | `cml`                                                      | One or more user-defined labels (delimited with commas).                                                                                       |
```

p.s. Labels are user-defined right? Are spaces allowed between commas?

Reply:
Yes, they're strings at the shell level, but they get automatically cast to ECMAScript types following the usual logic (?) 🙃  Anyhow, it feels awkward and your proposed solution looks much better.

Reply:
Yes, labels are user-defined

Reply:
Resolved with https://github.com/iterative/dvc.org/pull/2321/commits/ca804af226acd6100e23d25430d39b87277ef6f8